source/documentation/additional-information.html.md

```diff
@@ -8,14 +8,15 @@ description: Software developers, designers, product owners or business analysts
 
 If Making Tax Digital (MTD) customers are earning any type of income in addition to their self-employment and property income, they must report that to HMRC just as they would in their Self Assessment return. 
 
-><p>Currently, the endpoints for reporting additional income through software are included in the Individuals Income Received API. As of June 2024, the Individuals Income Received API will be deprecated and replaced with the following new APIs:</p><ul><li>Individuals Employments Income API</li> 
-<li>Individuals Dividends Income API</li>
-<li>Individuals Foreign Income API</li>
-<li>Individuals Insurance Policies Income API</li>
-<li>Individuals Pensions Income API</li>
-<li>Individuals Other Income API</li>
-<li>Individuals Savings Income API</li>
-<li>Individuals Capital Gains Income API</li></ul>
+><p>Currently, the endpoints for reporting additional income through software are included in the Individuals Income Received API. As of June 2024, the Individuals Income Received API will be deprecated and replaced with the following new APIs:</p><ul> 
+	<li><a href="/api-documentation/docs/api/service/individuals-employments-income-api/">Individuals Employments Income API</a></li>
+	<li><a href="/api-documentation/docs/api/service/individuals-dividends-income-api/">Individuals Dividends Income API</a></li>
+	<li><a href="/api-documentation/docs/api/service/individuals-foreign-income-api/">Individuals Foreign Income API</a></li> 
+	<li><a href="/api-documentation/docs/api/service/individuals-insurance-policies-income-api/">Individuals Insurance Policies Income API</a></li>
+	<li><a href="/api-documentation/docs/api/service/individuals-pensions-income-api/">Individuals Pensions Income API</a></li>
+	<li><a href="/api-documentation/docs/api/service/individuals-other-income-api/">Individuals Other Income API</a></li>
+	<li><a href="/api-documentation/docs/api/service/individuals-savings-income-api/">Individuals Savings Income API</a></li>
+	<li><a href="/api-documentation/docs/api/service/individuals-capital-gains-income-api/">Individuals Capital Gains Income API</a></li></ul>
 <p>These new APIs are now available for testing in Sandbox.</p>
 
```

Comment:
Did you have a particular reason for using HTML tags for this text block?

Can api documentation frontend repo resolve document references (without `http`) in <li> tags as opposed to Markdown links?

Reply:
Yes, I am using a block quote to highlight this section, and normal markdown links don't work with that. Slightly hacky. I am pretty sure the links will work, I'll check on QA